<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>File_Actions_window</class>
 <widget class="QWidget" name="File_Actions_window">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>406</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>406</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>406</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File actions</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Move to trash file</string>
   </property>
   <property name="whatsThis">
    <string>Move to trash file</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border: f;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/resource/img/white_theme/trash-512.png</normaloff>:/resource/img/white_theme/trash-512.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>305</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #252525;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>10</y>
     <width>501</width>
     <height>410</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: ;</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Rename file</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>Enter new file name</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>491</width>
      <height>221</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="title">
     <string>Edit time</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEdit_2">
       <property name="styleSheet">
        <string notr="true">border: 2px solid;
    border-radius: 7px;</string>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="styleSheet">
        <string notr="true">border: 2px solid;
    border-radius: 7px;</string>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Last modif</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEdit_3">
       <property name="styleSheet">
        <string notr="true">border: 2px solid;
    border-radius: 7px;</string>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Birth file </string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Open file </string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEdit_4">
       <property name="styleSheet">
        <string notr="true">border: 2px solid;
    border-radius: 7px;</string>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>MetaData file </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>305</y>
      <width>491</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Resize file</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QSpinBox" name="spinBox">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <number>1999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
